src/org/jitsi/impl/neomedia/MediaUtils.java

```diff
@@ -86,7 +87,7 @@
      */
     private static final Map<String, MediaFormat[]>
         rtpPayloadTypeStrToMediaFormats
-            = new HashMap<String, MediaFormat[]>();
+            = new ConcurrentHashMap<>();
```

Comment:
Like jmfEncodingToEncodings, rtpPayloadTypeStrToMediaFormats appears to be modified only during the static constructor of MediaUtils and, consequently, it seems unnecessary to use a ConcurrentHashMap.